include/utils/json.hpp

```diff
@@ -191,6 +191,14 @@ std::vector<T> GetArrayValue(const CaseInsensitiveObjectWrapper& object, const s
     return GetArrayValue<T>(object, key, [](const Poco::Dynamic::Var& value) { return value.convert<T>(); });
 }
 
+/**
+ * Stringifies json.
+ *
+ * @param json json object.
+ * @return std::string.
+ */
+std::string Stringify(const Poco::Dynamic::Var& json);
```

Comment:
This commit is taken from PR
https://github.com/aosedge/aos_core_common_cpp/pull/25

Will remove once that PR is merged